templates/tfengine/components/cicd/manual/README.md

```diff
@@ -95,6 +97,21 @@ Every new push to the Pull Request at the configured branches automatically
 triggers presubmit runs. To manually re-trigger CI jobs, comment `/gcbrun` in the
 Pull Ruquest.
 
+The deletion check optionally accepts a whitelist of resources to ignore. To
```

Comment:
Add more doc around in which format should users input the resource. Is it a terraform state address or resource name or something.